.editorconfig

```diff
@@ -2,44 +2,49 @@
 # .NET coding convention settings for EditorConfig
 # https://docs.microsoft.com/visualstudio/ide/editorconfig-code-style-settings-reference
 #
-# The .editorconfig comes from Roslyn and CoreFX repositories:
-# https://github.com/dotnet/corefx/blob/master/.editorconfig
+# This file comes from dotnet repositories:
+# https://github.com/dotnet/runtime/blob/master/.editorconfig
 # https://github.com/dotnet/roslyn/blob/master/.editorconfig
```

Comment:
With .editorconfig support in Omnisharp we could add more rules in our file too.

Reply:
@iSazonov Which rules would you like to see added?

Reply:
@iSazonov I have a new PR to reformat source code: #11681